### ISSUE TITLE
[nextgen-editor] handling of sub/sup (subscript/superscript) tags

### ISSUE THREAD

**Issue by rouben** [getgrav/grav-premium-issues#231]
I'm trying to edit the following Markdown sentence in the next-gen editor:
The formula for water is H<sub>2</sub>O. And this is a second sentence.

Typing the above directly into the next-gen editor simply strips out the `<sub>` tags. Note that this doesn't happen with tags that are on the next-gen editor toolbar, such as `<strong>` (bold text) or `<s>` (strikethrough).

The next thing I tried is using the raw HTML code snippet, however, the editor wraps the entire block in `<p>` tags, which makes it impossible to insert HTML code in the middle of a sentence. In other words, what I want to do is this:
```
The formula for water is H<div class="raw-html-embed"><sub>2</sub></div>O. And this is a second sentence.
```
what I end up with is this:
```
<p>
    The formula for water is H
</p>
<div class="raw-html-embed">
    <sub>2</sub>
</div>
<p>
    O. And this is a second sentence.
</p>
```

Is there a better way to do subscripts/superscripts with the next-gen editor? 

This is happening with the latest stable build of Grav (v1.7.30) and next-gen editor plugin (v1.1.11).

**Comment by rouben** [comment]
Any updates on this? Is there a way to extend the next-gen editor to support the subscript and superscript tags?

**Comment by w00fz** [comment]
Apologies, I haven’t gotten to this yet. I’ve been extremely busy, but It’s on my list.

**Comment by rouben** [comment]
Hi, just bumping this issue again, it has been over a month. I don't mean to rush, but am hoping to get an ETA on this, or a workaround we can use in the meantime. Thanks in advance!

**Comment by w00fz** [comment]
Apologies @rouben, 
I have a lot on my plate lately and trying to work through my never ending list of things to do.

I have the fix implemented right now, just going to go through some other ones for NextGen and hope to get a release out today.

Thank you for your patience

**Comment by w00fz** [comment]
This is now available in the v1.1.12 release.

Thanks!
